packages/jest-core/src/TestScheduler.ts

```diff
@@ -145,9 +145,13 @@ export default class TestScheduler {
           context.hasteFS,
           this._globalConfig.updateSnapshot,
           snapshot.buildSnapshotResolver(context.config),
+          context.config.testPathIgnorePatterns,
         );
 
         aggregatedResults.snapshot.filesRemoved += status.filesRemoved;
+        aggregatedResults.snapshot.filesRemovedList = [
```

Comment:
Don't need to spread, `concat` does not modify the original array

Reply:
Could you make the edit? And merge afterwards